src/cli.ts

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env node
+import 'reflect-metadata'
+import { render } from 'ink';
+import meow from 'meow';
+import { commandsRouter } from "./CommandsRouter";
+
+
+const cli = meow(`
+	Usage
+	  $ asyncapi
+	  
+	Commands
+		validate 
+			Options
+				-c --context  Your AsyncAPI specification
+				-w --watch Enable watchMode
```

Comment:
mark somehow that it is not yet implemented (same in readme)

Reply:
@jotamusik ?

Reply:
Sure! Didn't notice! Done 😋